### ISSUE TITLE
Update form error example to show correct accessibility settings

### ISSUE THREAD

**Issue by Blackbaud-TrevorBurch** [blackbaud/skyux2-docs#966]
We should be showing the use of `aria-describedby` when a form error is shown. We also should update the error message to use the new status indicator component in order to fix accessibility issues with the old css psuedo selector error icon.

However, we need to show smart usage of `id`s and so we should hold off on this until we finish and release an `id` creation service. That work is tracked at blackbaud/skyux-core#65

**Comment by blackbaud-johnly** [comment]
Added *On hold* label to visually highlight that this is waiting on the release of the `id` creation service, as Trevor said in the last paragraph of the description.
